en/components/grids_templates/column_hiding.md

```diff
@@ -492,7 +492,9 @@ Styles:
 * [Column Pinning](column_pinning.md)
 * [Column Resizing](column_resizing.md)
 * [Selection](selection.md)
+@@if (igxName !== 'IgxHierarchicalGrid') {
```

Comment:
![image](https://user-images.githubusercontent.com/12784974/53963176-6eb48400-40f5-11e9-9c2b-f8e7a8027c1c.png)

The blank lines caused by the **if** statements are reflected within a list.
In order to avoid this, we should probably use a one-liner for the **if** and its body.
**For example:** ```@@if (igxName !== 'IgxHierarchicalGrid') {* [Searching](search.md)}```